device.go

```diff
@@ -21,29 +23,74 @@ import (
 	grpcStatus "google.golang.org/grpc/status"
 )
 
+const (
+	driver9pType   = "9p"
+	driverBlkType  = "blk"
+	driverSCSIType = "scsi"
+)
+
+// SCSI variables
+var (
+	// Here in "0:0", the first number is the SCSI host number because
+	// only one SCSI controller has been plugged, while the second number
+	// is always 0.
+	scsiHostChannel = "0:0:"
+	sysClassPrefix  = "/sys/class"
+	scsiDiskPrefix  = filepath.Join(sysClassPrefix, "scsi_disk", scsiHostChannel)
+	scsiBlockSuffix = "block"
+	scsiDiskSuffix  = filepath.Join("/device", scsiBlockSuffix)
+	scsiHostPath    = filepath.Join(sysClassPrefix, "scsi_host")
+)
+
 type deviceDriversHandler func(device pb.Device, spec *pb.Spec) error
 
 var deviceDriversHandlerList = map[string]deviceDriversHandler{
-	"blk": blockDeviceHandler,
+	driverBlkType:  deviceBlkBlockDeviceDriverHandler,
+	driverSCSIType: deviceSCSIBlockDeviceDriverHandler,
 }
 
-func blockDeviceHandler(device pb.Device, spec *pb.Spec) error {
+func deviceBlkBlockDeviceDriverHandler(device pb.Device, spec *pb.Spec) error {
```

Comment:
`thisNameIsCrazyLongBlkBlockBlkBlahBleurk()` and a bit confusing (although precise :smile:)

How about changing the names to:

- `genericBlockDeviceHandler()`
- `scsiDeviceHandler()`

Reply:
Yes, I'm going to simplify them